SeaORM-TODO-App/book/src/tcp-server.md

```diff
@@ -18,23 +18,19 @@
    $ cargo add async-std --features attributes
    ```
 
-   This adds async-std `src/Cargo.toml` file
+   This adds async-std to `Cargo.toml` file
 
    ```toml
    [package]
-   name = "sea-orm-todo"
+   name = "todo-server"
    version = "0.1.0"
    edition = "2021"
    
-   # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-   
    [dependencies]
    + async-std = { version = "1.10.0", features = ["attributes"] }
    ```
 
-   
-
-2. Add `anyhow` crate for error handling
+1. Add `anyhow` crate for error handling
```

Comment:
Hey @charleschege, I was thinking to group steps into subheadings. Such as...

1. Setting up file structure for TCP Server
1. Installing necessary dependencies
1. Creating database in PostgreSQL
1. Configuring the environment variables